nekoyume/Assets/_Scripts/Extension/AvatarExtensions.cs

```diff
@@ -23,6 +23,24 @@ public static bool TryGetEquippedFullCostume(this AvatarState avatarState, out C
             return fullCostume is { };
         }
 
+        public static (bool isCharacterId, int id) GetPortraitId(
```

Comment:
Is `isCharacterId` a value that identifies a costume or armor? 
If yes, how about doing this? ⬇️ 
`var isCharacterId = TryGetEquippedFullCostume(avatarState, out var fullCostume);`

Reply:
The `characterId` makes some confuse so I removed that.